docs/migration-guide.rst

```diff
@@ -41,6 +41,10 @@ General
   <https://legacy-docs.rasa.com/docs/nlu/0.15.1/migrations/?&_ga=2.218966814.608734414.1560704810-314462423.1543594887#id1>`.
   These two files should be merged into a single file either named ``config.yml``, or passed via the ``--config`` parameter.
 
+- While not required, it's recommended that your project filenames and directory structure follow the
+  convention created by the `rasa init` command  (see :ref:`command-line-interface <create-a-new-project>`).
```

Comment:
```suggestion
  convention created by the ``rasa init`` command  (see :ref:`command-line-interface`).
```
In `.rst` files, the double `` has the same functionality as single ones in e.g. github. Single ones will italicize in rst